### DISCUSSION TITLE
figma-measure

### DISCUSSION THREAD

**Discussion by noel-reyes789** [ph1p/figma-measure#29] [General]
Hello! I recently downloaded the plug-in and up until today it was working great. All of the sudden it stopped working. I can still open it up but when I try to add measurements or a tooltip nothing happens. I tried closing it down and restarting both the plug-in and Figma but no luck. Any suggestions?

**Comment by ph1p** [comment]
I'll take a look at it. Please try to remove all your measurement groups. The plugin currently only uses one single group.
